csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "NuSpec.ReferenceGenerator": { "type": "build", "version": "2.0.0-beta-bld14" }
+  },
+  "frameworks": {
+    "netstandard1.0": {}
```

Comment:
I think the plan is that they just update the version of nuget - from [Oren's post](https://oren.codes/2016/06/23/portable-is-dead-long-live-netstandard/)

> With NuGet 2.12, netstandard is also supported in VS 2012 and 2013, so that there’s no further need to include a portable-* version and a netstandard version in the same package. Finally!